2024-06-03.mdx

```diff
@@ -0,0 +1,55 @@
+---
+date: 6-3-2024
+---
+
+### What's new with Xata
+
+**UI improvements**
+
+You might have noticed already, but we've been beautifying the UI in the past weeks. For example, check out this empty state page for a new database: 🤩
+
+![Empty state for the new database page](/images/2024-06-03-empty-state.png)
+
+Other UI improvements:
+
+* The Playground now supports running multiple statements at once.
+* The branch selector got an upgrade and now works better with looooong branch names.
+* Fixed using `now()` as a default value for date columns.
+
+
+**Postgres compatibility improvements**
+
+Also the past couple of weeks we've seen a ton of improvements to our compatibility with the Postgres wire protocol:
+  * Improved compatibility with more tools that use the Postgres wire protocol, including JetBrains DataGrip.
+  * Added support for several system info functions.
+  * Allow locking clauses in SELECT statements. For example, the following is now supported: `SELECT * FROM table1 WHERE name = 'John' FOR UPDATE`.
+  * Improved compatibility and fixed issues when running [Ghostfolio](https://github.com/ghostfolio/ghostfolio) with Xata.
+  * Reduced clutter and errors during pg_restore runs.
+  * Added support for `SHOW session_replication_role`.
+
+**Extensions!**
+
+We added support for the first extensions on shared clusters:
+
+* Added support for [pgvector](https://github.com/pgvector/pgvector). You can now use either pgvector or the Xata vector implementation based on Lucene.
+* Added support for the [uuid-ossp](https://www.postgresql.org/docs/current/uuid-ossp.html) extension. You can use its functions to generate different UUID flavours.
+
+**Dedicated clusters**
+
+We've worked on improving the stability and usability of dedicated clusters, currently in private alpha, but quickly moving towards public beta:
+
+* Better validation for cluster names to prevent a common source of errors.
+* Added support for terminating clusters.
+* Added support limited support for DO blocks. More improvements coming soon.
+
+**Data API**
+
+Also our API, more specifically the SQL over HTTP part, got an important update:
+
+* Support batching multiple SQL commands in a single request over the HTTP API.
+
+
+### Blog posts
```

Comment:
Depending on when this ships, we may have a tutorial from Teri tomorrow according to our content schedule. @CezzaineZ let us know if we need to update the changelog, no reason to rush if it's not there though.